### PR TITLE
Feature/linechart & barchart multiline

### PR DESCRIPTION
**Pull request by tarkus2000ua**

### CODE REVIEW

File: client/src/components/BarChart/BarChart.js [hunk 1]
```diff
@@ -110,19 +113,69 @@ function BarChart(props) {
       .on('mouseover', tip.show)
       .on('mouseout', tip.hide);
 
-    if (trendline.display && data.length) {
+      if (showDataPointsValues) {
+        debugger
```
Comment (YuriiStetsula) [inline client/src/components/BarChart/BarChart.js:117]: remove

File: client/src/components/BarChart/BarChart.js [hunk 2]
```diff
@@ -110,19 +113,69 @@ function BarChart(props) {
...
+          .append('text')
+          .attr('class', 'bar__value')
+          .attr('x', (a) => {
+            console.log(`band ${xScale.bandwidth()}`);
+            return xScale(a[XAxis.key]) + (2*index+1)*xScale.bandwidth()/(2*YAxis.key.length)})
+          .attr('y', (a) => yScale(a[YKey]) - 20)
+          .attr('text-anchor', 'middle')
+          .text((a) => `${a[YKey]}`);
+      }
+  debugger;
```
Comment (YuriiStetsula) [inline client/src/components/BarChart/BarChart.js:128]: remove

**Review by YuriiStetsula — CHANGES_REQUESTED** [review]

File: client/src/components/BarChart/BarChart.js (hunk 1 above)
Comment (tarkus2000ua) [reply client/src/components/BarChart/BarChart.js]: removed

File: client/src/components/BarChart/BarChart.js (hunk 2 above)
Comment (tarkus2000ua) [reply client/src/components/BarChart/BarChart.js]: removed

**Review by YuriiStetsula — APPROVED** [review]

**Pull request merged**